src/api_tracker.py

```diff
@@ -0,0 +1,68 @@
+import os
+import requests
```

Comment:
[minor] do we need request  just to make a post? if we already have it fine but o/w could just use core lib ...

Reply:
> @mbeilin small comments and one substantive suggested change. Do you want to get that in.
> 
> Coaching: think about what we would want to analyse be in google analytics e.g. just seeing a lot of uuids would be non optimal

In case of a lot of uuids you are right, it's going to be not useful, but we have BigQuery table names as resource id here,
e.g. EPD_202002, PCA_2014, so that makes sense I think.

Reply:
I still want them qualified by dataset - o/w they may not be unique ... (?)

Reply:
>Ahhh that's because we set the resource id to be table name elsewhere i think. I'm in two minds whether we should do that >but that's a refactoring so let's leave this as is for now and not make the change i'm suggesting.
So I leave it as is atm as per your suggestion ^^

Reply:
ok. let's do that.